coba/evaluators/online.py

```diff
@@ -235,8 +235,10 @@ def evaluate(self, environment: Optional[Environment], learner: Optional[Learner
                         predict_time = time.time()-start_time
                         if not batched:
                             ope_reward = sum(p*float(log_rewards.eval(a)) for p,a in zip(on_probs,log_actions))
+                            on_action, on_prob = sample_actions(log_actions, on_probs)
                         else:
                             ope_reward = [ sum(p*float(R.eval(a)) for p,a in zip(P,A)) for P,A,R in zip(on_probs,log_actions,log_rewards) ]
+                            on_action, on_prob = zip(*[sample_actions(actions, probs) for actions, probs in zip(log_actions, on_probs)])
                     else:
```

Comment:
How to do this for continuous actions?

Reply:
For continuous actions we just need to call `on_action,on_prob = predict(log_context, log_actions)[:2]`

Reply:
Maybe on line 246? I don't think we need to have separate processing for batched and non-batched. Man I hate all this batched logic. It's all here for neural network stuff we do where backpropagation with mini-batches can give huge gains in computation time.

Reply:
Tried to add support for continuous actions but struggle to make some tests pass, see below.